build.gradle

```diff
@@ -161,9 +161,9 @@ subprojects {
                 protobuf_plugin: 'com.google.protobuf:protobuf-gradle-plugin:0.8.0',
                 protobuf_util: "com.google.protobuf:protobuf-java-util:${protobufVersion}",
 
-                netty: 'io.netty:netty-codec-http2:[4.1.4.Final]',
-                netty_epoll: 'io.netty:netty-transport-native-epoll:4.1.4.Final' + epoll_suffix,
-                netty_tcnative: 'io.netty:netty-tcnative-boringssl-static:1.1.33.Fork19',
+                netty: 'io.netty:netty-codec-http2:[4.1.6.Final]',
+                netty_epoll: 'io.netty:netty-transport-native-epoll:4.1.6.Final' + epoll_suffix,
+                netty_tcnative: 'io.netty:netty-tcnative-boringssl-static:1.1.33.Fork23',
```

Comment:
We can use the older versions here, since we don't expose this dependency at all via Maven. If this doesn't work with Fork19, then we need to update SECURITY.md and make sure to notify users in our release notes. That said, when we bump the version, we can go to the newest, but if we don't need to bump it, we probably shouldn't, since that's what our users will do.